nwastdlib/graphql/extensions/error_handler_extension.py

```diff
@@ -93,10 +95,15 @@ def _add_extension(error: GraphQLError, key: str, value: Any) -> None:
 
 def _process(error: GraphQLError, to_error_type: Callable[[Exception | None], ErrorType]) -> GraphQLError:
     exc = error.original_error
+    error_type = to_error_type(exc)
     if isinstance(exc, HTTPStatusError):
         _add_extension(error, EXTENSION_HTTP_STATUS_CODE, {f"{exc.request.url}": exc.response.status_code})
     if not _has_extension(error, EXTENSION_ERROR_TYPE):
         _add_extension(error, EXTENSION_ERROR_TYPE, str(to_error_type(exc)))
+
+    if error_type == ErrorType.INTERNAL_ERROR and not (nwa_settings.DEBUG_PYCHARM or nwa_settings.DEBUG_PYCHARM):
+        error.message = "Internal Server Error"
```

Comment:
This is the easy way of masking everything internal that could be unintended error data for the fetching side.

I'm thinking of adding an extra `ErrorType` like `QueryError` which can be used to add data to the error msg.

could also specifically check for DB errors and only mask those.

Reply:
we can come back to this later if necessary.